/**:
  ros__parameters:
    height_threshold: 0.3
    slope_threshold: 15.0
    voxel_grid_size: .1
    ground_distance_threshold: .05
    bounding_box_start_point: [-0.5, 0.0, 0.5]
    bounding_box_length: 3.0
    bounding_box_width: 2.0
    bounding_box_height: 1.0
    epsilon_clustering_density: 0.5
    min_points_to_cluster: 20
